test/testSamples/MaterialX/grid_with_udims.usda

```diff
@@ -0,0 +1,121 @@
+#usda 1.0
+(
+    defaultPrim = "pPlane1"
+    metersPerUnit = 0.01
+    upAxis = "Y"
+)
+
+def Mesh "pPlane1" (
+    prepend apiSchemas = ["MaterialBindingAPI"]
+    kind = "component"
+)
+{
+    uniform bool doubleSided = 1
+    float3[] extent = [(-0.5, 0, -0.5), (0.5, 0, 0.5)]
+    int[] faceVertexCounts = [4, 4, 4, 4, 4, 4, 4, 4, 4]
+    int[] faceVertexIndices = [0, 1, 5, 4, 1, 2, 6, 5, 2, 3, 7, 6, 4, 5, 9, 8, 5, 6, 10, 9, 6, 7, 11, 10, 8, 9, 13, 12, 9, 10, 14, 13, 10, 11, 15, 14]
+    rel material:binding = </pPlane1/mtl/standardSurface2SG>
+    normal3f[] normals = [(0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0), (0, 1, 0)] (
+        interpolation = "faceVarying"
+    )
+    point3f[] points = [(-0.5, 0, 0.5), (-0.16666666, 0, 0.5), (0.16666669, 0, 0.5), (0.5, 0, 0.5), (-0.5, 0, 0.16666666), (-0.16666666, 0, 0.16666666), (0.16666669, 0, 0.16666666), (0.5, 0, 0.16666666), (-0.5, 0, -0.16666669), (-0.16666666, 0, -0.16666669), (0.16666669, 0, -0.16666669), (0.5, 0, -0.16666669), (-0.5, 0, -0.5), (-0.16666666, 0, -0.5), (0.16666669, 0, -0.5), (0.5, 0, -0.5)]
+    color3f[] primvars:displayColor = [(0.13320851, 0.13320851, 0.13320851)] (
+        customData = {
+            dictionary Maya = {
+                bool generated = 1
+            }
+        }
+    )
+    texCoord2f[] primvars:st = [(0.1, 0.1), (0.9, 0.1), (1.9, 0.1), (2.9, 0.1), (3.1, 0.1), (4.1, 0.1), (5.1, 0.1), (2.9, 0.9), (9.1, 0.1), (1.1, 2.1), (5.1, 5.1), (5.9, 0.9), (9.1, 0.9), (1.1, 2.9), (5.1, 5.9), (5.9, 5.9), (9.9, 0.9), (4.1, 0.9), (3.9, 0.9), (9.9, 0.1), (3.1, 0.9), (1.9, 2.9), (5.9, 5.1), (5.1, 0.9), (4.9, 0.9), (1.9, 2.1), (2.1, 0.9), (1.9, 0.9), (4.9, 0.1), (5.9, 0.1), (1.1, 0.9), (0.9, 0.9), (3.9, 0.1), (0.1, 0.9), (2.1, 0.1), (1.1, 0.1)] (
```

Comment:
Notice there are gaps in the UDIMS, we jump directly from tile 1006 to tile 1010 and 1022. This makes sure gaps are well handled, and that UDIMs at higher V values are working.

The UDIM atlas in this case will be 10x3 with 8 tiles loaded and 28 empty tiles. Not efficient, but that is how the current codebase loads UDIMs.